security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -17,6 +17,70 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
 
+
+[id="compliance-operator-release-notes-1-7-0_{context}"]
+== OpenShift Compliance Operator 1.7.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.7.0:
+
+* link:https://access.redhat.com/errata/RHBA-2025:3728-01[RHBA-2025:3728-01 - OpenShift Compliance Operator 1.7.0 bug fix and enhancement update]
+
+[id="compliance-operator-1-7-0-new-features-and-enhancements_{context}"]
+=== New features and enhancements
+
+* A `must-gather` extension is now available for the Compliance Operator installed on `aarch64`, `x86`, `ppc64le`, and `s390x` architectures. The `must-gather` tool provides crucial configuration details to Red Hat Customer Support and engineering. For more information, see xref:../../security/compliance_operator/co-support.adoc#compliance-must-gather_co-support[Using the must-gather tool for the Compliance Operator].
+
+* CIS Benchmark Support has been added to Compliance Operator 1.7.0. The profile supported is CIS OpenShift Benchmark 1.7.0. For more information, see (link:https://issues.redhat.com/browse/CMP-3081[*CMP-3081*])
+
+* Compliance Operator is now supported on `aarch64` architecture for CIS OpenShift Benchmarks 1.7.0 and FedRAMP Moderate Revision 4. For more information, see (link:https://issues.redhat.com/browse/CMP-2960[*CMP-2960*])
+
+* Compliance Operator 1.7.0 now supports OpenShift DISA STIG V2R2 profiles for OpenShift and RHCOS. For more information, see (link:https://issues.redhat.com/browse/CMP-3142[*CMP-3142*])
+
+* Compliance Operator 1.7.0 now supports deprecation of old, unsupported profile versions, such as deprecation of CIS 1.4 profiles, CIS 1.5 profiles, DISA STIG V1R1 profiles and DISA STIG V2R1 profiles. For more information, see (link:https://issues.redhat.com/browse/CMP-3149[*CMP-3149*])
+
+* With this release of Compliance Operator 1.7.0, the deprecation of older CIS and DISA STIG profiles mean that these older profiles will no longer be supported with the appearance of Compliance Operator 1.8.0. For more information, see (link:https://issues.redhat.com/browse/CMP-3284[*CMP-3284*])
+
+[id="compliance-operator-1-7-0-bug-fixes_{context}"]
+=== Bug fixes
+
+// OCPBUGS-10336: I have documented this as a new feature (above) and not as a bug.
+
+* Before this release, Compliance Operator would provide an unneeded remediation recommendation due to differences in filesystem structure for the `s390x` architecture. With this release, the Compliance Operator now recognizes the differences in filesystem structure and does not provide the misleading remediation. With this update, the rule is now more clearly defined. (link:https://issues.redhat.com/browse/OCPBUGS-33194[*OCPBUGS-33194*])
+
+* Previously, version detection did not work correctly on IBM cloud and Hypershift environments. With this release, Compliance Operator now checks `HostedCluster.Status.Version.History[0]` to report a correct result. (link:https://issues.redhat.com/browse/OCPBUGS-3953[*OCPBUGS-3953*])
```

Comment:
This bug should have been fixed many releases ago. I will remove this bug from the errata.